src/Provisioning.ts

```diff
@@ -203,9 +203,9 @@ export class Provisioner {
             return;
         }
         const results = await Promise.all(accounts.map(async (account) => {
-                const team = await this.main.datastore.getTeam(account.teamId)
-                return {team, slack_id: account.slackId};
-            })
+            const team = await this.main.datastore.getTeam(account.teamId)
```

Comment:
Surprised we don't enforce semi-colons? We should have a rule for that.

Reply:
I set semi to "error" now.